pynear/include/BKTree.hpp

```diff
@@ -3,13 +3,13 @@
 #include <optional>
 
 template <typename key_t, typename distance_t> class Metric {
-    public:
+public:
```

Comment:
Just access modifier identation here. I prefer it aligned to left. Let me know what you think. Some classes were changed due to clang-format config file change.

Reply:
I prefer them on the left as well.